deeprank/learn/DataSet.py

```diff
@@ -1217,6 +1218,7 @@ def map_one_molecule(self, fname, mol=None, angle=None, axis=None):
         try:
             target = mol_data.get('targets/' + self.select_target)[()]
         except Exception:
+            target = None
             logger.exception(f'No target value for: {fname} - not required for the test set')
```

Comment:
Make more sense to change `logger.exception` to `logger.warning`?

Reply:
Same :)